programs/system/src/invoke/sol_compression.rs

```diff
@@ -61,9 +59,7 @@ pub fn decompress_lamports<
         None => return err!(SystemProgramError::DeCompressLamportsUndefinedForDecompressSol),
     };
 
-    transfer_lamports(&sol_pool_pda, &recipient, lamports)?;
-
-    Ok(())
```

Comment:
does it make any difference to not emit Ok(()) here

Reply:
It's cleaner not to